packages/oncokb-frontend-commons/src/components/OncoKB.tsx

```diff
@@ -0,0 +1,270 @@
+                    OncoKbCardDataType.DX,
+                    OncoKbCardDataType.PX,
+                ].map(dataType => (
+                    <AnnotationIcon
+                        type={dataType}
+                        tooltipOverlay={tooltipContent(
+                            dataType,
+                            props,
+                            handleFeedbackOpen
+                        )}
+                        indicator={props.indicator}
+                        availableDataTypes={props.availableDataTypes}
+                    />
+                ))}
+            </>
+        );
+    } else {
+        // workaround: use content padding value to draw an empty icon when there is no indicator data.
+        // this is to keep the icon alignment consistent with the rest of the column.
+        // ideally we should implement grouped columns to avoid these kind of workarounds
+        // (see https://github.com/cBioPortal/cbioportal/issues/8723)
+        return <i style={{ paddingRight: props.contentPadding }} />;
+    }
+}
+
+function singleAnnotationIcon(
+    props: IOncoKbProps,
+    handleFeedbackOpen: () => void
+) {
+    return (
+        <span className="oncokb-content" style={{ display: 'inline-flex' }}>
+            <AnnotationIconWithTooltip
+                tooltipOverlay={tooltipContent(
+                    findDefaultDataTypeForTooltip(
+                        props.usingPublicOncoKbInstance,
+                        props.indicator,
+                        props.availableDataTypes
+                    ),
+                    props,
+                    handleFeedbackOpen
+                )}
+                icon={
+                    <CompactAnnotationIcon
+                        indicator={props.indicator}
+                        availableDataTypes={props.availableDataTypes}
+                        usingPublicOncoKbInstance={
+                            props.usingPublicOncoKbInstance
+                        }
+                    />
+                }
+            />
+        </span>
+    );
+}
+
+function tooltipContent(
+    type: OncoKbCardDataType,
+    props: IOncoKbProps,
+    handleFeedbackOpen: () => void
+): JSX.Element {
+    return (
+        <OncoKbTooltip
+            type={type}
+            usingPublicOncoKbInstance={props.usingPublicOncoKbInstance}
+            hugoSymbol={props.hugoGeneSymbol}
+            geneNotExist={props.geneNotExist}
+            isCancerGene={props.isCancerGene}
+            indicator={props.indicator || undefined}
+            handleFeedbackOpen={
+                props.disableFeedback ? undefined : handleFeedbackOpen
+            }
+        />
+    );
+}
+
+export const OncoKB: React.FunctionComponent<IOncoKbProps> = (
+    props: IOncoKbProps
+) => {
+    const [showFeedback, setShowFeedback] = React.useState(false);
+    const [
+        tooltipDataLoadComplete,
+        setTooltipDataLoadComplete,
+    ] = React.useState(false);
```

Comment:
I'm not sure what the purpose of this is. I also asked Onur about it a while ago

Reply:
Looks like not used anymore. This might be used originally for the loading gif but I am not sure. We need to go over the change history to see why and when this became unused.